common/views/components/Discussion/Discussion.tsx

```diff
@@ -0,0 +1,49 @@
+import { FunctionComponent } from 'react';
+import { HTMLString } from '../../../services/prismic/types';
+import { Person } from '../../../model/people';
+import PrismicHtmlBlock from '../PrismicHtmlBlock/PrismicHtmlBlock';
+
+type Props = {
+  title: string | null;
+  discussion: {
+    contributor: Person | null;
+    text: HTMLString | null;
+  }[];
+};
+
+const Discussion: FunctionComponent<Props> = ({ title, discussion }: Props) => {
+  const textWithContributorNameAdded = discussion.map(section => {
+    const contributor = `${section?.contributor?.name}:`;
+    return (
+      section.text &&
+      section.text.map((text, i) => {
+        if (i === 0 && section.contributor) {
+          return {
+            type: text.type,
+            text: `${contributor} ${text.text}`,
+            spans: [
+              {
+                start: 0,
+                end: contributor.length,
```

Comment:
Would this be `+1` because of the space we add before the text? Or is this a `0` index?

Reply:
it's 0 index